js/main.js

```diff
@@ -133,10 +133,10 @@ async function main() {
         }
         break;
 
-      case "/collection.html":
+      case "/collection":
+        setupCollectionControls();
```

Comment:
do we need to change the other places with `*.html`? I think it doesn't break anything if we don't update all the places, but messages.js has some similar switch cases set up to this that would be good to make consistent

All the hardcoded urls in each page's html also still have an href of `*.html`

Reply:
Thanks for the heads-up! Changed the routing in messages.js too.

If we turn on the Pretty URLs setting in Netlify, it searches for urls in the html files and automatically rewrites them without the html extension, so nothing has to be changed in that aspect.